core/src/jvm_bridge/comet_task_memory_manager.rs

```diff
@@ -0,0 +1,62 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use jni::{
+    errors::Result as JniResult,
+    objects::{JClass, JMethodID},
+    signature::{Primitive, ReturnType},
+    JNIEnv,
+};
+
+use crate::jvm_bridge::get_global_jclass;
+
+/// A DataFusion `MemoryPool` implementation for Comet, which delegate to the JVM
+/// side `CometTaskMemoryManager`.
```

Comment:
The comment looks not correct as `CometMemoryPool` is the one implementing DataFusion `MemoryPool`. Maybe this should be moved to `CometMemoryPool`?

Reply:
Yea let me move it.